diff --git a/screen-CB3eVjAV.js b/screen-CB3eVjAV.js
index 52d048274d2321612fd31078b90252ab0acfa06c..678c20f1f84e7a769443b012332dd97df9855352 100644
--- a/screen-CB3eVjAV.js
+++ b/screen-CB3eVjAV.js
@@ -5979,7 +5979,7 @@ class Pe {
           r.close(o);
         }
       });
-      t.cleanup(() => i.close()), t.run(() => {
+      t.cleanup(() => { if (i.state !== "closed") i.close(); }), t.run((t) => {
         const o = t.get(this.#r);
         o && i.configure(o);
       }), t.run((o) => {
@@ -6106,7 +6106,7 @@ class Pe {
     if (!Hi)
       for (const a of i) {
         if (!a.startsWith(r)) continue;
-        const c = "prefer-hardware", u = {
+        const c = "no-preference", u = {
           codec: a,
           width: s.width,
           height: s.height,
